incubator/java-openliberty/templates/binary/pom.xml

```diff
@@ -0,0 +1,156 @@
+<?xml version='1.0' encoding='utf-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
```

Comment:
Hmm...arguable, but maybe we should drop the parent altogether.  Without the local dev mode, I'm not sure it's buying us anything.

Reply:
OK..one point I hadn't thought was the idea that the stack author could "blacklist" certain dependencies.    This would prevent them from getting built into the Docker image.   Maybe we should say that's valid then, reintroduce the parent, and adjust validate.sh to not blow up for our binary template?

Reply:
parent pom stanza re-introduced, no longer skipping the check for it if binary template